metadata.rb

```diff
@@ -4,6 +4,6 @@
 license          'All rights reserved'
 description      'Installs/Configures gate-sso'
 long_description IO.read(File.join(File.dirname(__FILE__), 'README.md'))
-version          '0.4.7'
+version          '0.4.9'
```

Comment:
Any specific reasons for not going with 0.4.8

Reply:
There is already a PR raised with 0.4.8, so we went with 0.4.9.

Reply:
Please use `0.4.8` instead

Reply:
changed the version to `0.4.8`.